pkg/standalone/standalone.go

```diff
@@ -48,8 +49,64 @@ const (
 	DaprRedisContainerName = "dapr_redis"
 )
 
+func isInstallationRequired(installLocation, requestedVersion string) bool {
+	destDir := daprDefaultLinuxAndMacInstallPath
+	if installLocation == "" {
+		if runtime.GOOS == daprWindowsOS {
+			destDir = daprDefaultWindowsInstallPath
+		}
+	}
+
+	daprdBinaryPath := filepath.Join(destDir, daprRuntimeFilePrefix) //e.g. /usr/local/bin/daprd or c:\\daprd
+
+	//First time install?
```

Comment:
Nit: please add space between // and First :)

Reply:
fixed :) (along with other comments as well)